sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Upstream is wrong then: https://www.odoo.com/documentation/8.0/reference/guidelines.html